lib/logger.js

```diff
@@ -0,0 +1,42 @@
+const winston = require('winston')
+const { createLogger, format, transports } = winston
+const { combine, timestamp, label, printf } = format
+
+// Custom logging format
+const customFormat = printf(({ level, message, label, timestamp }) => {
+  return `${timestamp} [${label || '-'}] ${level}: ${message}`
+})
+
+// Custom combined logging format:
+const customCombinedFormat = module =>
+  combine(
+    format.colorize({ all: true }),
+    label({ label: module }),
+    timestamp(),
+    customFormat
+  )
+
+// Custom transports:
+const customTransports = () => [new transports.Console()]
+
+// Container to provide different pre-configured loggers
+const logContainer = new winston.Container()
+
+// Example: Pre-configured module logger with special properties
+logContainer.add('ZWC', {
```

Comment:
What's `ZWC` ?

Reply:
It's the short representation of "ZwaveClient" for shorter logging statements. It just shows that it may be different to the (possibly longer) filename.
It's also one of the assumptions we did not yet talk about and I'd like to get some feedback about what is expected.

Reply:
I think ZJS2M it's more appropiated as we also have a zwave client module. 

Modules are: 

- `store`: the json store module
- `app`: everything that is in `app.js`
- `apis`: to log api requests (will replace morgan)
- `zwave`: zwave client
- `mqtt`: mqtt client